recipes/tutorials/plugins_example.ipynb

```diff
@@ -1,825 +1,911 @@
 {
```

Comment:
<p>This info is not updated : the plugin is not about IBAN and Currency anymore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/909/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
done